.github/workflows/test_linux.yaml

```
@@ -0,0 +1,45 @@
+name: Linux CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - nightly
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```

Or does v2 have any advantages over v3?